src/components/AttestedClaimVerificationView/AttestedClaimVerificationView.tsx

```diff
@@ -68,27 +85,51 @@ class AttestedClaimVerificationView extends React.Component<Props, State> {
   private static readonly BLOCK_CHAR: string = '\u2588'
 
   private buildClaimPropertiesView(attestedClaim: IAttestedClaim): JSX.Element {
-    const { cType } = this.props
-
-    const propertyNames: string[] = Object.keys(
-      attestedClaim.request.claimHashes
-    )
+    const { cType } = this.state
+
+    let properties: Array<{ key: string; label: string; value: string }> = []
+    if (cType) {
+      const propertyNames = Object.keys(cType.cType.schema.properties)
+      properties = propertyNames.map(propertyName => {
+        const label = getCtypePropertyTitle(propertyName, {
+          // TODO: What the hell? Why do we have two so similar types for a ctype with metadata?
```

Comment:
It is something we had to do because of typing. Its something I wanted to change for a while.

It adds new methods or did previously to the `ctypeWithMetadata` in the demo-client. Something we decided not to change a while back and been neglectic.

Reply:
Though, I don't know the exact state of it at the moment.